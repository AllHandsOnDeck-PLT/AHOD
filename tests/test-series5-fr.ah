series<string> deck
series<string> hand
int i
main:
{
    /* This is a comment */
    deck = ["d","e","f","g"]
    hand = ["a","b","c"] /* this is another comment */
    hand.push(deck.pop()) /* deck.pop() gives 'e' */
    hand.push(deck.pop()) /* deck.pop() gives 'f' */
    
    /* hand = ["a", "b", "c", "f", "g"] */
    /* deck = ["d","e"] */

    do PRINT("hand is now:")
    for  (i = 0; i < hand.size(); i = i + 1):
    {
    do PRINT(hand[i])
    }
    do PRINT("deck is now:")
    for  (i = 0; i < deck.size(); i = i + 1):
    {
    do PRINT(deck[i])
    }
}
