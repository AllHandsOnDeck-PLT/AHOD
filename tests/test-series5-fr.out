hand is now:
a
b
c
g
f
deck is now:
d
e
